src/wizard/wizard.scss

```diff
@@ -102,15 +106,15 @@
       }
     }
 
-    > .iui-wizards-step-current {
-      > .iui-wizards-step-title {
+    .iui-wizards-step.iui-wizards-step-current {
+      .iui-wizards-step-title {
         @include themed {
           color: t(iui-color-foreground-positive);
         }
         font-weight: $iui-font-weight-semibold;
       }
 
-      > .iui-wizards-step-circle {
```

Comment:
why from all of them this was removed?

Reply:
I was trying to be less specific so if someone decided to wrap something around these elements it would still work.  I've reverted it back to try to keep the changes minimal.